src/data/finsupp/basic.lean

```diff
@@ -411,6 +427,46 @@ lemma support_update_ne_zero [decidable_eq α] (h : b ≠ 0) :
 
 end update
 
+/-! ### `indicator` -/
```

Comment:
Do you think this stuff could go in `data/finsupp/indicator` rather than making this file longer?

Reply:
Fully agree! I already tried such a split over at #11109 but it didn't go well. I'm gonna create `data.finsupp.indicator` and hopefully migrate stuff from`.basic` over there.

Reply:
Sure, feel free to do that in a separate splitting PR